pom.xml

```diff
@@ -132,13 +132,14 @@
                 <snapshots.repository.name>Sonatype Nexus snapshot repository</snapshots.repository.name>
                 <snapshots.repository.url>https://oss.sonatype.org/content/repositories/snapshots</snapshots.repository.url>
                 <!-- Build configuration for release -->
+                <image.repository>gchq/palisade-</image.repository>
```

Comment:
This value is set above so does not need to be set here as well.

Reply:
Done